packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -1754,7 +1738,7 @@ describeCompat("GC data store tombstone tests", "NoCompat", (getTestObjectProvid
 					mockLogger,
 				);
 				// Send an op from dataStore2's root DDS and request dataStore2 so that the DDS is loaded and the
-				// handle to dataStore3 is parsed resulting in a notification to GC of reference to dataStore3.
+				// handle to dataStore3 is parsed. This should not result in any reference notification to GC.
```

Comment:
Oops looks like this should have been updated previously, right?

Reply:
Yeah, the behavior isn't changed with this PR.